sdk/python/feast/feature_store.py

```diff
@@ -2283,45 +2226,12 @@ def serve(
     ) -> None:
         """Start the feature consumption server locally on a given port."""
         type_ = type_.lower()
-        if self.config.go_feature_serving and self._go_server:
-            # Start go server instead of python if the flag is enabled
-            self._lazy_init_go_server()
-            enable_logging = (
-                self.config.feature_server
-                and self.config.feature_server.feature_logging
-                and self.config.feature_server.feature_logging.enabled
-                and not no_feature_log
-            )
-            logging_options = (
-                self.config.feature_server.feature_logging
-                if enable_logging and self.config.feature_server
-                else None
+        if type_ != "http":
+            raise ValueError(
+                f"Python server only supports 'http'. Got '{type_}' instead."
             )
-            if type_ == "http":
-                self._go_server.start_http_server(
-                    host,
-                    port,
-                    enable_logging=enable_logging,
-                    logging_options=logging_options,
-                )
-            elif type_ == "grpc":
-                self._go_server.start_grpc_server(
-                    host,
-                    port,
-                    enable_logging=enable_logging,
-                    logging_options=logging_options,
-                )
-            else:
-                raise ValueError(
-                    f"Unsupported server type '{type_}'. Must be one of 'http' or 'grpc'."
-                )
-        else:
-            if type_ != "http":
```

Comment:
this probably still needs to exizt?

Reply:
It's there actually - look up at line 2229 in the new version.